### ISSUE TITLE
valid_docs false positive on () closures (Void)

### ISSUE THREAD

**Issue by BluMist** [realm/SwiftLint#580]
Not a big deal as we can switch to Void, but it was not obvious why the warning occurred.

**no warning:**
`/**
     Description

```
 - parameter callback: description
 */
func method(callback: (Bool) -> Void) {
    callback(true)
}`
```

**warning:**
`/**
     Description

```
 - parameter callback: description
 */
func method(callback: (Bool) -> ()) {
    callback(true)
}`
```

**Comment by jpsim** [comment]
Thanks for reporting this.

**Comment by marcelofabri** [comment]
Closing this since we removed this rule.
